guides/common/modules/con_disaster-recovery-with-active-and-passive-project-server-and-external-storage.adoc

```diff
@@ -0,0 +1,6 @@
+[id="disaster-recovery-with-active-and-passive-{project-context}-server-and-external-storage"]
+= Disaster recovery with active and passive {ProjectServer} and external storage
+
+In this scenario, you maintain two {ProjectServer}s and you store critical data externally on network-attached storage (NAS).
```

Comment:
`In this scenario` refers to what's described in the section heading, right? Section body is then dependent on section heading, which I see seldom. If there are no guidelines against this, looks good.